test/js/test.js

```diff
@@ -200,11 +230,39 @@ function sendAgreement(termsURL) {
         console.log("Couldn't POST agreement back to server, aborting.");
         process.exit(1);
       } else {
-        getChallenges({domain: state.domains.pop()});
+          if (state.nextTests) {
+              rotateAccountKey();
+          } else {
+              getChallenges({domain: state.domains.pop()});
+          }
       }
     });
 }
 
+function rotateAccountKey() {
+    // Make a new account key
+    var oldAcme = state.acme;
+    makeAccountKeyPair("new-account-key.pem", function() {
+        var payload = JSON.stringify({
+            oldKey: oldAcme.privateKey.publicKey,
+            newKey: state.acme.privateKey.publicKey,
+            account: state.registrationURL,
+        }, null, 2)
+        var signed = cryptoUtil.generateSignature(state.acme.privateKey, new Buffer(payload), oldAcme.nonces.shift());
```

Comment:
https://ietf-wg-acme.github.io/acme/#rfc.section.6.3.1
> The inner JWS is NOT REQUIRED to have a “nonce” parameter. The server MUST ignore any value provided for the “nonce” header parameter.

Looking at `generateSignature`, it seems you can simply omit the none parameter.